liberty-maven-plugin/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -13,5 +13,18 @@
         <ignore />
       </action>
     </pluginExecution>
+    <pluginExecution>
+      <pluginExecutionFilter>
+        <goals>
+          <goal>export-plugin-config</goal>
+        </goals>
+      </pluginExecutionFilter>
+      <action>
+        <execute>
+          <runOnIncremental>false</runOnIncremental>
+          <runOnConfiguration>true</runOnConfiguration>
```

Comment:
After "mvn clean" in Eclipse, the user needs to do a "maven project update" to re-gen this xml file.  It doesn't matter if the project has no java code.